### ISSUE TITLE
[Bug]: css content alt text is not included in aria-snapshots

### ISSUE THREAD

**Issue by spike-rabbit** [microsoft/playwright#35737]
### Version

1.52.0

### Steps to reproduce

1. clone https://github.com/spike-rabbit/playwright-css-content-alt-text
2.  Install deps: npm i
3. Start server: npm run serve
4.  Run tests: npm run test

### Expected behavior

The tests in the repository should succeed, e.g. playwright should recognize the alt text of the css content attribute.

### Actual behavior

The test fails as playwright does not recognize the alt text of the css content attribute.

### Additional context

CSS allows specifying an alternative text for the content attribute like
```css
.with-content {
 content: 'url(...)' / "Alt text"
}
```

See: https://developer.mozilla.org/en-US/docs/Web/CSS/content

This text is also properly visible when using the A11y tools of the respective browser.
But playwright aria-snapshots do not include this alt text.

Not sure if separate problem, the axe-tools integration is also not aware of this.

### Environment

```shell
System:
    OS: Linux 6.13 Fedora Linux 41 (KDE Plasma)
    CPU: (16) x64 AMD Ryzen 7 PRO 5850U with Radeon Graphics
    Memory: 13.00 GB / 30.17 GB
    Container: Yes
  Binaries:
    Node: 22.13.1 - ~/.nvm/versions/node/v22.13.1/bin/node
    Yarn: 1.22.22 - ~/.nvm/versions/node/v22.13.1/bin/yarn
    npm: 11.1.0 - ~/.nvm/versions/node/v22.13.1/bin/npm
  IDEs:
    VSCode: 1.99.3 - /usr/bin/code
  Languages:
    Bash: 5.2.32 - /usr/bin/bash
  npmPackages:
    @playwright/test: ^1.52.0 => 1.52.0
```
